tests/listsAndDropDowns.spec.ts

```diff
@@ -9,76 +9,76 @@ test.beforeEach(async ({ page }) => {
 
 test.describe("Lists and Drop Downs", async () => {
   test("Validate selected pet types from the list", async ({ page }) => {
+    const type1Input = page.locator("#type1");
+    const petTypeSelect = page.locator("#type");
     await page.getByText("George Franklin").click();
 
     await expect(page.locator(".ownerFullName")).toHaveText("George Franklin");
 
-    await page.getByRole("button", { name: "Edit Pet" }).nth(0).click();
+    const petLeoSection = page.locator("app-pet-list", { name: "Leo" });
+    await petLeoSection
+      .getByRole("button", { name: "Edit Pet" })
+      .first()
+      .click();
 
     await expect(page.locator("h2")).toHaveText("Pet");
 
     await expect(page.locator("#owner_name")).toHaveValue("George Franklin");
 
-    await expect(page.locator("#type1")).toHaveValue("cat");
+    await expect(type1Input).toHaveValue("cat");
 
-    const dropDownMenu = page.locator("#type");
-    await dropDownMenu.click();
+    await petTypeSelect.click();
```

Comment:
You had a good name for the locator, but changed it to a bad name :)

Reply:
improved :)